packages/react/src/ErrorBoundary.en.mdx

```diff
@@ -58,3 +58,44 @@ const Example = () => {
   )
 }
 ```
+
+## useErrorBoundary (experimental feature)
+
+We can reset ErrorBoundary without props using useErrorBoundary.reset in ErrorBoundaryFallback.
+
+```tsx
+const ErrorBoundaryFallback = () => {
+  const errorBoundary = useErrorBoundary()
+
+  return <button onClick={errorBoundary.reset} />
+}
+
+const Example = () => (
+  <ErrorBoundary fallback={ErrorBoundaryFallback}>
+    <Error />
+  </ErrorBoundary>
+)
+```
+
+We can use useErrorBoundary.setError in ErrorBoundaryFallback to make ErrorBoundary aware of the Error without throw.
+
+```tsx
+const SetErrorAfterFetch = () => {
+  const errorBoundary = useErrorBoundary()
```

Comment:
In fact, when using the `useErrorboundary`, I used it put errors that occur other than rendering into the errorboundary. I think it is good to added in the document

[reference](https://slash.page/libraries/react/error-boundary/src/errorboundary.i18n/#useerrorboundary)
>React's [error boundary](https://reactjs.org/docs/error-boundaries.html) component cannot catch an error occurred in event handlers, asynchronous code like setTimeout, etc.
useErrorBoundary is useful to deliver an error to the nearest error boundary in anywhere.

Reply:
Cool👍, Could you add it more please after merging this Pull Request